applications: serial_lte_modem: SLM server fixes. (#12590)
nrfconnect/sdk-nrf: applications/serial_lte_modem/doc/slm_testing.rst

         :class: highlight

         **AT#XTCPSVR=1,**\ *1234*
         #XTCPSVR: 0,"started"
         OK

         **AT#XTCPSVR?**
         #XTCPSVR: 0,-1,1
         OK

   #. Run the :file:`client_tcp.py` script to start sending data to the server.

   #. Observe that the server accepts the connection from the client and receives the first packets.
      Read the information about the current state again.

      .. parsed-literal::
         :class: highlight

         #XTCPSVR: "*IP address*","connected"

         #XTCPDATA: 13
         Hello, TCP#1!
         #XTCPDATA: 13
         Hello, TCP#2!

         **AT#XTCPSVR?**
         #XTCPSVR: 0,1,1
         OK

>    #. Send responses and receive the rest of the data. Client disconnects after receiving the last response.

Reviewer: ```suggestion
   #. Send responses and receive the rest of the data.
      Client disconnects after receiving the last response.
```

TLS server
==========

> The TLS server role is currently not supported without :file:`overlay-native_tls.conf` configuration file.

Reviewer: ```suggestion
The TLS server role is currently only supported when using the :file:`overlay-native_tls.conf` configuration file.
```
